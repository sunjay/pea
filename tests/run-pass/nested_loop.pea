fn main() {
    let x = 1;
    while x <= 3 {
        let y = 1;
        while y <= 4 {
            print!(x);
            print!(b" * ");
            print!(y);
            print!(b" = ");
            println!(x*y);

            y += 1;
        }

        x += 1;
    }
}
