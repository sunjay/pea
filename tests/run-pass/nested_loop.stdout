1 * 1 = 1
1 * 2 = 2
1 * 3 = 3
1 * 4 = 4
2 * 1 = 2
2 * 2 = 4
2 * 3 = 6
2 * 4 = 8
3 * 1 = 3
3 * 2 = 6
3 * 3 = 9
3 * 4 = 12
